lib/jobs/pdu-node-relations.js

```diff
@@ -0,0 +1,142 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = pduRelationsJobFactory;
+di.annotate(pduRelationsJobFactory, new di.Provide('Job.Catalog.PduRelations'));
+di.annotate(pduRelationsJobFactory, new di.Inject(
+    'Job.Base',
+    'Services.Waterline',
+    'Logger',
+    'Util',
+    'Promise',
+    'Assert'
+));
+
+function pduRelationsJobFactory(
+    BaseJob,
+    waterline,
+    Logger,
+    util,
+    Promise,
+    assert
+) {
+
+    var logger = Logger.initialize(pduRelationsJobFactory);
+
+    /**
+     *
+     * @param {Object} [options]
+     * @constructor
+     */
+    function PduRelationsJob(options, context, taskId) {
+        PduRelationsJob.super_.call(this, logger, options, context, taskId);
+
+        this.nodeId = context.target || options.nodeId;
+        assert.isMongoId(this.nodeId);
```

Comment:
Don't do this, The RackHD will support multi backends database,  then the nodeId will be set as uuid not MongoId.